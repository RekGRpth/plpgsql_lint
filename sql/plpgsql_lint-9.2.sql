/home/pavel/src/postgresql/contrib/plpgsql_lint/sql/plpgsql_lint-9.0.sql