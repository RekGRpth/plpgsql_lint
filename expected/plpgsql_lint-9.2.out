/home/pavel/src/postgresql/contrib/plpgsql_lint/expected/plpgsql_lint-9.1.out